CalibFormats/CaloObjects/src/CaloSamples.cc

```diff
@@ -3,14 +3,13 @@
 #include <math.h>
 #include <iostream>
 
-const int CaloSamples::MAXSAMPLES;
-
```

Comment:
`MAXSAMPLES` is no longer used in the `CaloSamples` class itself, so I don't think that line is needed.

I left the const defined in the header because it's used somewhere else (ECAL code maybe?) and I didn't want to deal with it in this PR.